### PR TITLE
Server deployment

### PR DESCRIPTION
**Pull request by vinjana**
A server deployment script to be run via cron on the GHGA website server. The script gets the commit hash of the MASTER head here in this repo, checks whether it is among the hashes towards the local master in a deployment-repo. If the hash is new, it pulls (force!) the repo to local, builds with jekyll and deploys into the server directory. Logging is predefined to console, syslog **AAAAAND Slack**.

### CODE REVIEW

File: src/ghgaSiteUpdateDaemon.py [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+
+
+logger = logging.getLogger("ghga_updater")
+# slackHandlers = list(filter(lambda h: h.get_name() == "slack", logger.handlers))
+# if len(slackHandlers) == 0:
+#     print("Omitting logging to Slack", file=stderr)
+# else:
+#     print("Logging to Slack", file=stderr)
+#     slackFormatter = jsonlogger.JsonFormatter(rename_fields={'message': 'text'})
+#     slackHandlers[0].setFormatter(slackFormatter)
```
Comment (KochTobi) [inline src/ghgaSiteUpdateDaemon.py:168]:
```suggestion
```

File: src/update-current-branch-and-deploy-ghga-site.sh [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+cd "$siteRepo"
+echo "############################################################################" >> /dev/stderr
+echo "## Is the right branch checked out? PLEASE CHECK                          ##" >> /dev/stderr
+echo "############################################################################" >> /dev/stderr
+git branch --list
+sleep 10
+git pull
+bundle exec jekyll build
+rsync -avuc --delete "$siteRepo/_site/" /var/www/html/
+chmod a+rX -R /var/www/html/
```
Comment (KochTobi) [inline src/update-current-branch-and-deploy-ghga-site.sh:15]: Is this file needed?

**Review by KochTobi — APPROVED** [review]

**Comment by vinjana** [comment]
Will probably never be used.
